docs.json

```diff
@@ -43,6 +43,301 @@
   },
   "navigation": {
     "tabs": [
+      {
```

Comment:
Can you move these to a yaml file and generate the docs.json instead? If there's a feature missing from the generator we can work on that. All of this looks pretty standard.

Reply:
done